### PR TITLE
Deployment Fixes

### PR DESCRIPTION
**Pull request by mzappitello**
* skip process_trip_updates if no new events records to prevent a key error.
* add better logging to main ingestion function.
* fix oversight by adding files to archive file list in gtfs static ingestion.
* use correct filepath for aws ssl cert file.
* wrap file_list_from_s3 in a try/except block.

### CODE REVIEW

File: py_gtfs_rt_ingestion/lib/s3_utils.py [hunk 1]
```diff
@@ -71,8 +65,24 @@ def strip_timestamp(fileobject: str) -> str:
         filepath = pathlib.Path(fileobject)
         return filepath.name[:20]
 
-    filepaths.sort(key=strip_timestamp)
-    return filepaths
+    try:
```
Comment (mzappitello) [inline py_gtfs_rt_ingestion/lib/s3_utils.py:68]: wrap this in a try except and log it better.

File: py_gtfs_rt_ingestion/lib/postgres_utils.py [hunk 2]
```diff
@@ -63,9 +63,7 @@ def get_local_engine(echo: bool = False) -> sa.engine.Engine:
             # set the ssl cert path to the file that should be added to the
             # lambda function at deploy time
             db_ssl_cert = os.path.abspath(
-                os.path.join(
-                    os.path.abspath(__file__), "..", "..", "aws-cert-bundle.pem"
-                )
+                os.path.join("/", "usr", "local", "share", "amazon-certs.pem")
```
Comment (mzappitello) [inline py_gtfs_rt_ingestion/lib/postgres_utils.py:66]: fix the bad path.

File: py_gtfs_rt_ingestion/lib/convert_gtfs.py [hunk 3]
```diff
@@ -73,3 +35,42 @@ def get_tables(self) -> List[Tuple[str, pyarrow.Table]]:
...
+                    filename_prefix = gtfs_filename.replace(".txt", "").upper()
+                    tables.append((filename_prefix, table))
+
+            process_logger.log_complete()
+            self.archive_files.append(filename)
+            return tables
+        except Exception as exception:
+            process_logger.log_failure(exception)
+            self.error_files.append(filename)
+            return []
```
Comment (mzappitello) [inline py_gtfs_rt_ingestion/lib/convert_gtfs.py:76]: same function, but add files to error and archive.

File: py_gtfs_rt_ingestion/ingest.py [hunk 4]
```diff
@@ -82,24 +82,29 @@ def validate_environment() -> None:
         )
 
 
-@schedule.repeat(schedule.every().hour.at(":05"))
-def batch_and_ingest() -> None:
+@schedule.repeat(schedule.every(5).minutes)
```
Comment (mzappitello) [inline py_gtfs_rt_ingestion/ingest.py:85]: i made this more frequent to find errors easier. maybe its ok to change it back now?

File: py_gtfs_rt_ingestion/ingest.py [hunk 5]
```diff
@@ -82,24 +82,29 @@ def validate_environment() -> None:
...
     files = file_list_from_s3(
         bucket_name=os.environ["INCOMING_BUCKET"],
         file_prefix=DEFAULT_S3_PREFIX,
     )
 
+    total_filecount = 0
+
     for converter in ingest_files(files):
+        sub_process_logger = ProcessLogger("process_converter")
+        sub_process_logger.log_start()
```
Comment (mzappitello) [inline py_gtfs_rt_ingestion/ingest.py:106]: log these.

File: performance_manager/lib/rt_trip_updates.py [hunk 6]
```diff
@@ -404,21 +404,23 @@ def process_trip_updates(db_manager: DatabaseManager) -> None:
             sizes = {}
 
             new_events = get_and_unwrap_tu_dataframe(paths)
-            sizes["new_events_unwrapped_size"] = new_events.shape[0]
+            new_events_records = new_events.shape[0]
```
Comment (mzappitello) [inline performance_manager/lib/rt_trip_updates.py:407]: ryans change.

**Comment by rymarczy** [comment]
LGTM

**Pull request merged**